src/components/Articles.jsx

```diff
@@ -27,43 +27,94 @@ const Articles = ({ loggedIn }) => {
 		setSelectedTopic(e.target.name);
 	};
 
+	const sortArticles = (e) => {
```

Comment:
This isn't DRY as it repeats the functionality in the useEffect. It would make sense to hold the sort value in state and add this as a dependency to the useEffect

Reply:
I'd also suggest doing this with the topic rather than filtering in the FE - if the topic changes I would expect a refresh of the data so we get the most up to date articles